style.css

```diff
@@ -338,7 +340,232 @@ textarea {
   gap: 50px;
 }
 
+/* Style of popup added through javascript Mobile */
+.work-info {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transition: 2000ms ease-in-out;
+  transform: translate(-50%, -50%) scale(1);
+  height: 100%;
+  width: 100%;
+  z-index: 2;
+}
+
+.work-info > * {
+  flex: 0;
+}
+
+.cancel-btn {
+  position: absolute;
+  top: 4.1%;
+  right: 7%;
+  background: none;
+  border: none;
+}
+
+.cancel-btn img {
+  width: 35px;
+  height: 35px;
+  visibility: hidden;
+}
+
+.work-image {
+  box-sizing: border-box;
+  border-radius: 22px;
+}
+
+.work-image > img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.work-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 2.2em;
+  color: #172b4d;
+  text-align: left;
+  line-height: 1.2em;
+  margin-top: 0.5em;
+}
+
+.tech-list {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+  gap: 1em;
+  margin: 1em 0;
+}
+
+.list1 {
+  background: #ebf0ee;
+  color: #3a4a42;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  border-radius: 0.25em;
+  padding: 0.8em;
+}
+
+.description {
+  color: #344563;
+  text-align: left;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1em;
+  line-height: 1.5em;
+}
+
+.work-buttons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.wrk-btn {
+  display: flex;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  align-self: center;
+  text-align: center;
+  background: #36b37f;
+  font-weight: 600;
+  font-size: 16px;
+  letter-spacing: 2px;
+  padding: 0.9em;
+  color: white;
+  border-radius: 3px;
+}
+
+.wrk-btn > img {
+  margin-left: 1.3em;
+  margin-top: -0.2em;
+}
+
+.Desktop-img {
+  display: none;
+}
+
 @media (min-width: 768px) {
+  /* Pop design added through javaScript css */
+  .blurr {
+    filter: blur(10px);
+  }
+
+  .Mobile-img {
+    display: none;
+  }
+
+  .Desktop-img {
+    display: initial;
+  }
+
+  .work-info {
+    display: grid;
+    height: 90%;
+    width: 70%;
+    grid-template-columns: 1fr 1fr 1fr 30px;
+    grid-template-rows: 30px 55% 10% 10% 25%;
+  }
```

Comment:
- [ ] Please kindly check the `About me` section of your app, the content seem to be overlapping the `Get My Resume` button. Kindly fix it as requested by the previous reviewer.
### **Take A Look**
![abouGet](https://user-images.githubusercontent.com/55596970/188961983-9c240c4b-03b0-4569-bac0-ea65ff336fa0.png)

Reply:
Honestly, I don't know what's happening. It appears ok from my side. but I updated some changes. i hope its fixed
![Screenshot_2022-09-07_21-29-40](https://user-images.githubusercontent.com/106544844/188971593-005ba466-27cf-4368-973b-626df309e013.png)

Reply:
It's good now 
### **Optional** 

- [ ] Please you can add functionality to make the button work